Vagrantfile

```diff
@@ -63,8 +63,9 @@ Vagrant.configure("2") do |config|
   config.vm.provision "docker" do |d|
     d.build_image "/vagrant/docker",
       args: "-t devopsloft/devopsloft"
-    d.run "devopsloft/devopsloft",
-      args: "-p 80:80"
+    d.run "web",
```

Comment:
I'm not 100% sure thy vagrant is building the image
isn't the image run *inside* the VM?

Reply:
the image is built by Vagrant and than the container based on the image is spinned by Vagrant inside the VM